library/Exceptions/LanguageCodeException.php

```diff
@@ -13,14 +13,22 @@
 
 namespace Respect\Validation\Exceptions;
 
-class LanguageCodeException extends ValidationException
+/**
+ * @author Danilo Benevides <danilobenevides01@gmail.com>
+ * @author Emmerson <emmersonsiqueira@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ */
+final class LanguageCodeException extends ValidationException
 {
+    /**
+     * {@inheritdoc}
+     */
     public static $defaultTemplates = [
         self::MODE_DEFAULT => [
-            self::STANDARD => '{{name}} must be a valid language',
+            self::STANDARD => '{{name}} must be a valid language code set for ISO 639',
```

Comment:
Perhaps `{{name}} must be a valid ISO 639 language code`?

Reply:
Saw it on LanguageCode.md, but I can remove if needed.